lib/ljglibs/cdefs/gio.moon

```diff
@@ -280,6 +280,8 @@ ffi.cdef [[
     GApplicationCommandLine *cmdline,
     int *argc);
 
+  gchar * g_application_command_line_get_cwd (GApplicationCommandLine *cmdline);
```

Comment:
Missing a `const` here.

Reply:
Thanks for reviewing. I've just added the missing const.